packages/vue/src/components/organisms/SfTabs/_internal/SfTab.vue

```diff
@@ -16,22 +16,27 @@
         />
       </div>
     </slot>
+
     <div class="sf-tabs__content">
       <div v-if="isActive" class="sf-tabs__content__tab">
-        <!--@slot Default. Here you should pass your tab content -->
-        <slot />
+        <SfScrollable :max-content-height="maxContentHeight">
```

Comment:
@AdamPawlinski  we need props `tabMaxContentHight`, `tabShowText` and `tabHideText`

Reply:
@AdamPawlinski what did you think about set these props on SfTabs? Because you set it for all Tab at Tabs :) Check provide/inject at Vue.js

Reply:
That's okay. I'll use it :)